packages/_server/src/config/cspellConfig.ts

```diff
@@ -599,6 +599,25 @@ interface CSpellSettingsPackageProperties extends CSpellSettings {
 
     /**
      * @scope resource
+     * @description
+     * @markdownDescription
+     * List of RegExp patterns or Pattern names to exclude from spell checking.
+     * The default RegExp flag is `//gi`
+     * 
+     * **GUI Examples**
+     * 
+     * Everyline is a RegExp pattern or a Pattern names - comment
+     * 
+     * - `/["href"]/`: exclude the word "href", for its frequent use in HTML.
+     * - `/\b[A-Z]{1,5}\b/g`: exclude full-caps acronyms with 1-5 length.
+     * 
+     * **JSON Example**
+     * 
+     * `.vscode/settings.json` file, `cSpell.ignoreRegExpList`
+     * accepts an array of strings with vscode escaped chars, where
+     * `\` escape to `\\`, like in `.code-snippets` files.
+     * 
+     * - `/\b[A-Z]{1,5}\b/g` in GUI becomes `/\\b[A-Z]{1,5}\\b/g` in JSON.
```

Comment:
I think it reads better if you give the JSON example followed by the GUI example.

Reply:
Changed to a table